KeyVaultCa.Core/EstConfiguration.cs

```diff
@@ -7,5 +7,7 @@ public class EstConfiguration
         public string IssuingCA { get; set; }
 
         public int CertValidityInDays { get; set; } = 365;
```

Comment:
is that still needed? Won't this be taken from appsettings.json?

Reply:
sorry, I wasn't specific enough. I was referring to the initialization

Reply:
Initially, I wanted to consider this property as optional and had a default value assigned, but with the current way of configuring, it should not be needed anymore. This value would anyway be overridden by what is specified in other settings.